src/webview_provider/newLineagePanel.ts

```
@@ -268,7 +268,12 @@ export class NewLineagePanel implements LineagePanelView {
   }
 
   private async addModelColumnsFromDB(project: DBTProject, node: NodeMetaData) {
-    const columnsFromDB = await project.getColumnsOfModel(node.name);
+    let columnsFromDB;
+    try {
+      columnsFromDB = await project.getColumnsOfModel(node.name);
+    } catch (error) {
+      return false;
+    }
```

If you ignore the exception here it will be lost. This seems wrong.